.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,70 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ main ]
+  schedule:
+    - cron: '00 6 * * 1'
```

Comment:
When this is run independently of a `merge` or `pull_request` event, where do the results go? Is there a pre-configured alerting email?

Reply:
Hi Sean, thanks for reviewing!

If a scheduled scan turns up a new finding, the alert will show up in the repo's Security tab, under "Code scanning alerts" ([instructions](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/managing-code-scanning-alerts-for-your-repository#viewing-the-alerts-for-a-repository)).

Unlike the code scanning alerts per PR which can be seen publicly, the repo's code scanning alerts page is only accessible to those with write permissions to the repo (more info [here](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#access-requirements-for-security-features)).

It's not clear from [GitHub's docs on alerting](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#dependabot-alerts-notification-options) yet whether code scanning alerts will result in email notifications in the same way as Dependabot does for insecure dependencies.